### PR TITLE
Corrected script for some item #17

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

**Review by Atemo — APPROVED** [review]
Look good!

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -132213,6 +132927,58 @@ Body:
...
+      if (eaclass()&EAJL_THIRD && BaseJob == Job_Taekwon) {
+         bonus bAspdRate,10;
+         bonus bMatkRate,25;
+         bonus2 bSubClass,Class_Boss,10;
+         bonus2 bMagicAtkEle,Ele_Dark,25;
+         if (.@r>=7) {
+            .@val = 50;
+            if (.@r>=9)
+                .@val += 50;
+         bonus2 bIgnoreDefRaceRate,RC_All,.@val;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:132979]: Thanks!

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -130871,6 +131550,41 @@ Body:
...
+      if (eaclass()&EAJL_THIRD && BaseJob == Job_Star_Gladiator) {
+         bonus bAtkRate,50;
+         bonus bSPDrainValue,2;
+         bonus2 bSubClass,Class_Boss,10;
+         autobonus "{ bonus bBaseAtk,150; }",20,3000,BF_WEAPON;
+         if (.@r >=7) {
+            .@val = 50;
+            if (.@r>=9)
+                .@val = 50;
+         bonus2 bIgnoreDefRaceRate,RC_All,.@val;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:131585]: Thanks!

**Comment by Atemo** [comment]
Thanks again!

**Pull request merged**